extensions/awt/runtime/src/main/java/io/quarkus/awt/runtime/JDKSubstitutions.java

```diff
@@ -30,34 +38,159 @@ private void verifyPFB(ByteBuffer bb) throws FontFormatException {
     }
 }
 
+/**
+ * AWT source code does not take into account a situation where "java.home" does not
+ * exist. It looks for default fonts in conf/fonts and lib dirs. It is O.K. if there are
+ * none as then system fonts are used instead. If the directory structure as such does not exist,
+ * the code path fails though.
+ *
+ * We create a dummy "java.home" in "java.io.tmpdir" and we set it at a reasonable place via
+ * substitution.
+ */
 @TargetClass(className = "sun.awt.FontConfiguration")
 final class Target_sun_awt_FontConfiguration {
     @Alias
     protected static String osVersion;
     @Alias
     protected static String osName;
 
-    /**
-     * AWT source code does not take into account a situation where "java.home" does not
-     * exist. It looks for default fonts in conf/fonts and lib dirs. It is O.K. if there are
-     * none as then system fonts are used instead. If the directory structure as such does not exist,
-     * the code path fails though.
-     *
-     * We create a dummy "java.home" in "java.io.tmpdir" and we set it at a reasonable place via
-     * substitution.
-     */
     @Substitute
     protected void setOsNameAndVersion() {
         final Path javaHome = Path.of(System.getProperty("java.io.tmpdir"), "quarkus-awt-tmp-fonts");
         try {
-            Files.createDirectories(Path.of(javaHome.toString(), "conf", "fonts"));
-            Files.createDirectories(Path.of(javaHome.toString(), "lib"));
+            System.setProperty("java.home", javaHome.toString());
+            osName = System.getProperty("os.name", "unknown");
+            osVersion = System.getProperty("os.version");
+            Files.createDirectories(javaHome.resolve("lib"));
```

Comment:
Is there no need for `conf/fonts` to be present any more? If so please avoid the class comment as well.